terraform/rds_to_s3_lambda.tf

```diff
@@ -65,15 +65,15 @@ resource "aws_lambda_function" "rds_to_s3_etl_lambda" {
 
   package_type  = "Image"
   architectures = ["x86_64"]
-  image_uri = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c14-trendgineers-rds-to-s3-ecr@sha256:80ce6b0839817fc7abcabc740003ce68ecf934b9bca2dcdcb9e229376804308e"
+  image_uri = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c14-trendgineers-rds-to-s3-ecr@sha256:18b75dc0397d28f7d0b6898876b1ce66845ea118cc31536c4a8ad866eadd944f"
```

Comment:
do these need to be protected variables

Reply:
These should be fine as is, users would need access to the current protected variables to make use of the image_uri.